jackmangosa1/certificate-manager
# KAN-26-Setup the project

#### PR description
This PR sets up the initial structure and configuration for a React application built from scratch.

#### Ticket / Jira Link
- [KAN-26](https://task-force-pro-dccs.atlassian.net/browse/KAN-26)

#### Checklist:
- [x] Configured Babel for transpiling modern JavaScript (ES6+) and JSX.
- [x] Configured Webpack for bundling the application.
- [x] Added development dependencies such as ESLint and Prettier for code quality and formatting.
- [x] Installed additional Webpack plugins and loaders for handling CSS, images, and other assets.

#### How To Test Changes

- Clone the repository: Clone the repository:
  ```bash
  git clone https://github.com/jackmangosa1/certificate-manager.git
- Navigate to the project directory.
- Install dependencies:   
```bash
   npm install
```
- Run the application locally:
```bash
npm start
```
- Verify that ESLint:
```bash
 npm run lint
```
- Verify prettier:
```bash
npm run format
```
